README.md

```diff
@@ -8,44 +8,28 @@ Improving the way that the data can be viewed and analysed/disseminated, so that
 
 There is an expectation that each file of this repository template is concise and self-documented.
 
+This repo contains the following:
+- `docs/adr` Contains the ADR 'decisions'
+- `infrastructure/` Contains infrastructure related Terraform templates for deploying the app.
+- `scripts/` Various scripts including githooks, functions etc.
+- `tests/` Contains tests related to the app
+
 ## Table of Contents
 
 - [uec cm](#uec-cm)
   - [Table of Contents](#table-of-contents)
-  - [Installation](#installation)
-    - [Prerequisites](#prerequisites)
-  - [Usage](#usage)
   - [Architecture](#architecture)
     - [Diagrams](#diagrams)
     - [Configuration](#configuration)
   - [Contributing](#contributing)
   - [Contacts](#contacts)
   - [Licence](#licence)
 
-## Installation
```

Comment:
The ticket specifically calls out the need for "installation and prerequisites need to be updated" - not to be removed. Can this be updated to be relevant to the project instead of deleted

Reply:
I think the installation and prerequisites should go into the specific Readme's. 

For example, src/frontend/Readme should contain installation and prerequisites for the React app. The infrastructure/Readme should contain infrastructure specific installation and prerequisites sections and so on. So it makes more sense to have generic information in the root readme.

Reply:
At the minute there is no infrastructure/Readme. If as part of this ticket you want to create it instead of in the root read me that works of me. 

But if we are separating the readme instead of moving the src/frontend/Readme to the root we can have the installation and prerequisites in the src/frontend/Readme. ie. from the ticket "Include project-specific details and instructions to make it more useful for contributors or remove it in favour of the root README"

It might be useful to look at other NHS that use react etc to see their readme structure.

Reply:
There is already a readme in infrastructure: https://github.com/NHSDigital/uec-cm/blob/main/infrastructure/README.md

I will remove unnecessary information from the root readme and we are adding setup information of the react app in the frontend Readme.